tests/unit10.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { test, expect } from '@playwright/test';
+import { WEB_URL, USER_EMAIL, USER_PASSWORD, USER_NAME } from '../utils/env';
+
+test('Verify login with valid credentials', async ({ page }) => {
+  await page.goto(`${WEB_URL}/auth/login`);
```

Comment:
let's move goto to beforeEach method? https://playwright.dev/docs/api/class-test#test-before-each

Reply:
Ooops, you can ignore this comment. Or split tests into several files and use beforeEach with those test starting with home page